test/python/pulse/test_transforms.py

```diff
@@ -88,40 +100,37 @@ def test_align_post_u3(self):
                 self.assertEqual(time, 10)
 
     def test_multi_acquire(self):
-        """Test that an error is raised if multiple acquires occur on the same channel."""
-        sched = pulse.Schedule(name='fake_experiment')
-        sched = sched.insert(0, Play(self.short_pulse, self.config.drive(0)))
-        sched = sched.insert(4, Acquire(5, self.config.acquire(0), MemorySlot(0)))
-        sched = sched.insert(10, Acquire(5, self.config.acquire(0), MemorySlot(0)))
-        with self.assertRaises(PulseError):
-            transforms.align_measures([sched], self.inst_map)
+        """Test that the last acquire is aligned to if multiple acquires occur on the
+        same channel."""
+        sched = pulse.Schedule()
+        sched.insert(0, Play(self.short_pulse, self.config.drive(0)), inplace=True)
+        sched.insert(4, Acquire(5, self.config.acquire(0), MemorySlot(0)), inplace=True)
+        sched.insert(20, Acquire(5, self.config.acquire(1), MemorySlot(1)), inplace=True)
+        sched.insert(10, Acquire(5, self.config.acquire(0), MemorySlot(0)), inplace=True)
+        aligned = transforms.align_measures([sched], self.inst_map)
+
+        ref = pulse.Schedule()
+        ref.insert(0, Play(self.short_pulse, self.config.drive(0)), inplace=True)
```

Comment:
with `align_all=True` shouldn't this be shifted?

Reply:
No, as this is a drive pulse not a measurement. We can only align all across schedules. Aligning within a schedule would affect the intraschedule timing and we can't know what drive/control channels are meant to do.